tests/ui/invalid_pymethod_proto_args.stderr

```diff
@@ -0,0 +1,5 @@
+error: Expected 1 arguments, got 0
+ --> tests/ui/invalid_pymethod_proto_args.rs:8:8
+  |
+8 |     fn __truediv__(&self) -> PyResult<()> {
+  |        ^^^^^^^^^^^
```

Comment:
Do you think we should include the receiver in these counts? E.g. the above becomes `Expected 2 arguments, got 1` ?

Reply:
Hmm, I was thinking about it but it might be a bit of a burden because we can also have "py" argument and take it into account which might be confusing?

Reply:
Ah I see. Can we add a test with `py` to confirm the behavior is correct in that situation too?

Other than that, I'm happy for us to leave the error message like this for now. We can always make adjustments later after we get user feedback 👍